<!DOCTYPE html>
<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Karczewski lab</title><!-- Global site tag (gtag.js) - Google Analytics -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-18414693-1"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'UA-18414693-1');
    </script>


    <!-- Bootstrap CSS CDN -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
    <!-- Our Custom CSS -->
    <link type="text/css" rel="stylesheet" href="style.css">
    <!-- jQuery CDN - Slim version (=without AJAX) -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <!-- Popper.JS -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
    <!-- Bootstrap JS -->
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>

    <!-- Font Awesome JS -->
    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/solid.js" integrity="sha384-tzzSw1/Vo+0N5UhStP3bvwWPq+uvzCMfrN1fEFe+xBmv1C/AtVX5K0uZtmcHitFZ" crossorigin="anonymous"></script>
    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/fontawesome.js" integrity="sha384-6OIrr52G08NpOFSZdxxz1xdNSndlD4vdcf/q2myIUVO0VsqaGHJsB0RaBE01VTOY" crossorigin="anonymous"></script>
</head>

<body data-spy="scroll" data-target="#sidebar">
<div class="wrapper">
    <!-- Sidebar -->
    <nav id="sidebar">
        <!--        <div class="sidebar-header">-->
        <!--            <h3>konradjk</h3>-->
        <!--        </div>-->
        <ul class="list-unstyled components nav flex-column">
            <p>Welcome!</p>
            <li class="nav-item">
                <a class="nav-link active" href="#about">About</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#research">Research</a>
                <!--                <a href="#pageSubmenu" data-toggle="collapse" aria-expanded="false" class="dropdown-toggle">Research</a>-->
                <!--                <ul class="collapse list-unstyled" id="pageSubmenu">-->
                <!--                    <li>-->
                <!--                        <a href="#">Page 1</a>-->
                <!--                    </li>-->
                <!--                    <li>-->
                <!--                        <a href="#">Page 2</a>-->
                <!--                    </li>-->
                <!--                    <li>-->
                <!--                        <a href="#">Page 3</a>-->
                <!--                    </li>-->
                <!--                </ul>-->
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#philosophy">Lab philosophy</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#publications">Publications</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#press">Press</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#public_projects">Public Projects</a>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="#contact">Contact</a>
            </li>
        </ul>
    </nav>
    <!-- Page Content -->
    <div id="content">
        <div id="topnav">
            <nav class="navbar navbar-expand-lg navbar-light bg-light">
                <div class="container-fluid">
                    Karczewski Lab
                    <button class="btn btn-dark d-inline-block d-lg-none ml-auto" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                        <i class="fas fa-align-justify"></i>
                    </button>
                    <div class="collapse navbar-collapse" id="navbarSupportedContent">
                        <ul class="nav navbar-nav ml-auto">
                            <li class="nav-item">
                                <a class="nav-link" href="#research">Research</a>
                            </li>
                            <li class="nav-item active">
                                <a class="nav-link" href="#people">People</a>
                            </li>
                            <li class="nav-item active">
                                <a class="nav-link" href="#philosophy">Philosophy</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#publications">Publications</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#public_projects">Public Projects</a>
                            </li>
                            <li class="nav-item">
                                <a class="nav-link" href="#contact">Contact</a>
                            </li>
                        </ul>
                    </div>
                </div>
            </nav>
        </div>
        <section class="jumbotron text-center" id="about">
            <div class="container">
              <h1 class="jumbotron-heading">Karczewski lab</h1>
              <p class="lead text-muted">Interpreting genetic variation using large genomic datasets.</p>
<!--              <p>-->
<!--                <a href="#" class="btn btn-primary my-2">Main call to action</a>-->
<!--                <a href="#" class="btn btn-secondary my-2">Secondary action</a>-->
<!--              </p>-->
            </div>
        </section>
        <div id="inner_content">
            <div class="row">
                <div class="col-sm-8">
                    <div id="research">
                        <h3>Research</h3>
                        <p>
                            At the Karczewski lab at Massachusetts General Hospital and the Broad Institute, our research is focused on assembling and analyzing massive public datasets of genetic variation,
                            and developing novel strategies using these to aid in the interpretation of putative disease variants,
                            in order to better distinguish causal disease variants and improve our understanding of human biology.
                        </p>
                        <p>
                            Broadly speaking, we are interested in genome sequencing and its future role in our daily lives.
                            With the age of rapidly decreasing sequencing costs, it is not difficult to imagine an age where
                            personal genetic information plays an important role in medicine and daily life.
                            We are constantly discovering more and more of the genetic basis of diseases, but much work has
                            yet to be done in fully explaining the genetic components of disease and other phenotypes.
                        </p>
                        <h4>A non-exhaustive list of projects</h4>
                        <ul>
                            <li>
                                Interpreting genetic variants:
                                <ul>
                                    <li>
                                        leveraging the demographic history of human populations
                                    </li>
                                    <li>
                                        using first principles of the genome (<a href="https://github.com/konradjk/loftee">LOFTEE</a>)
                                    </li>
                                </ul>
                            </li>
                            <li>
                                Understanding the function of genes and the genome:
                                <ul>
                                    <li>
                                        through the lens of <a href="https://www.nature.com/articles/s41586-020-2308-7">natural selection</a>
                                    </li>
                                    <li>
                                        using <a href="https://www.medrxiv.org/content/10.1101/2021.06.19.21259117">rare variant associations</a>
                                    </li>
                                    <li>
                                        using functional genomics data
                                    </li>
                                    <li>
                                        using directional perturbation of genes
                                    </li>
                                </ul>
                            </li>
                        </ul>
                    </div>
                    <div id="people">
                        <h3>People</h3>
                        <h5>Konrad Karczewski</h5>
                        <h6>Assistant Professor</h6>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/KonradKarczewski.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                            <p>
                                I am a computational biologist working on interpreting genetic variation from large-scale datasets.
                                As exome and genome data grow to massive scales, the methods and analytical frameworks
                                need to scale at the same rate, and interpreting results from large-scale analyses is
                                an exciting challenge. My group builds methods to interpret genetic variation, to learn
                                about the function of human genes and the regulation of the genome as a whole.
                                I earned a B.A. in Molecular Biology from Princeton University and a Ph.D. in Biomedical
                                Informatics from Stanford University.
                            </p>
                            <p>
                                <a href="https://twitter.com/konradjk?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">@konradjk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
<a class="github-button" href="https://github.com/konradjk" aria-label="Follow @konradjk on GitHub">@konradjk</a>
                            </p>
                            </div>
                        </div>
                        <br/>
                        <h4>Computational Scientists</h4>
                        <h5>Bram Gorissen</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/BramGorissen.png" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Bram is a computational scientist in the Karczewski Lab at the Broad Institute. He received his PhD in operations research from Tilburg University. He was an assistant professor at VU Amsterdam where he created and gave courses on big data and on convex optimization. He developed <a href="https://3142.nl/nymph/">Nymph</a>, the fastest exact inverse planning algorithm for radiation therapy, which is currently used by Massachusetts General Hospital. Previously he worked in the Fishell lab on, i.a., discovering enhancers.
                                </p>
                            </div>
                        </div>
                        <br/>
                        <h4>Fellows</h4>
                        <h5>Jeremy Guez</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/JeremyGuez.jpeg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Jeremy is a Postdoctoral Research Fellow in Karczewski Lab at ATGU and the Broad Institute of MIT and Harvard. He holds a BS degree in Biology from Sorbonne University, a MS in Genetics from Paris Saclay University and a PhD in Population Genetics from the French National Museum of Natural History. During his PhD, he studied the impacts of cultural transmission of reproductive success (CTRS) on population genetics and worked on its inference from human genomics data using machine learning methods. He is interested in research at the intersection of medical genomics, population genetics and machine learning.
                                </p>
                            </div>
                        </div>
                        <h5>Siwei Chen</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/SiweiChen.jpeg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Siwei is a Postdoctoral Research Fellow in Dr. Benjamin Neale's lab at ATGU and the Broad Institute of Harvard and MIT. Prior to joining the ATGU, Siwei earned her Ph.D. in Computational Biology from Cornell University, where her dissertation focused on identifying and interpreting disease mutations in the human protein interactome. Siwei is widely interested in studying the genetic basis of human diseases and is currently working on whole-exome/genome genetic variation analysis to understand how genetic risk translates into biological mechanisms.
                                </p>
                            </div>
                        </div>
                        <h5>Julia Sealock</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/JuliaSealock.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Julia is a research fellow who joined ATGU in May 2022. She completed a PhD in Human Genetics from Vanderbilt University where she studied the relationship between depression polygenic scores and lab traits extracted from electronic health records. As a research fellow, she is working on expanding rare variant association studies across diverse populations. In the Neale lab, she is leading the SCHEMA 3.0 analyses for schizophrenia and in the Karczewski lab she is applying GeneBass to non-European populations in the UKBiobank.
                                </p>
                            </div>
                        </div>
                        <h5>Mohamed El-Brolosy</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/MohamedElbrolosy.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Mohamed El-Brolosy is a junior fellow of the Harvard Society of Fellows, and a visiting scientist at Jonathan Weissman's lab at the Whitehead Institute.  Prior to that, he obtained his PhD at the Max Planck Institute for Heart and Lung Research with Didier Stainier. Mohamed is broadly interested in genetic robustness. Having discovered the phenomenon of Transcriptional Adaptation to mutations in his graduate studies, he is hoping to understand its potential therapeutic implications, how it affects the landscape of genetic diseases, and how different kinds of mutations within a given gene can lead to different phenotypic outcomes.
                                </p>
                            </div>
                        </div>
                        <br/>
                        <h4>Graduate Students</h4>
                        <h5>Hannah Jacobs</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/HannahJacobs.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Hannah is a PhD student at MIT's Department of Biology, in the Karczewski lab at the Broad Institute and in the Burge lab at MIT. She holds a BA in Biochemistry from Wellesley College. While at Wellesley she worked as an undergraduate researcher in Phil Sharp’s lab at MIT for two years studying microRNAs. In her PhD she is interested in combining population genetics and RNA biology to discover functional variants that underlie complex diseases.
<!--                                    <div class="badge-base LI-profile-badge" data-locale="en_US" data-size="medium" data-theme="light" data-type="VERTICAL" data-vanity="hannah-jacobs" data-version="v1">-->
                                        <a class="badge-base__link LI-simple-link" href="https://www.linkedin.com/in/hannah-jacobs/">LinkedIn</a>
<!--                                    </div>-->
                                </p>
                            </div>
                        </div>
                        <br/>
                        <h4>Associate Computational Scientists</h4>
                        <h5>Wenhan Lu</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/WenhanLu.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Wenhan is an Associate Computational Biologist in the Karczewski and Neale Labs at the Broad Institute. She earned her BS Degree in Mathematics at Nankai University and MS Degree in Biostatistics at Yale University. She is interested in developing statistical methods to reveal the underlying messages from large-scale biomedical data, as well as building pipelines for the quality control of large data and data virtualization.
                                </p>
                            </div>
                        </div>
                        <h5>Sophie Parsa</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/SophieParsa.png" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Sophie is a computational associate in the Karczewski and Martin labs. She is interested in creating deep learning models from genomic sequencing data. She completed a BS in Computer Science and a MS in Biomedical Informatics, both at Stanford. She is passionate about the intersection of computer science and medicine.
                                </p>
                            </div>
                        </div>
                        <br/>
                        <h4>Associate software engineers</h4>
                        <h5>Riley Grant</h5>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/RileyGrant.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Riley is an associate software engineer working on the Genome Aggregation Database’s (gnomAD’s) Web Browser features and UI. His interests lie in the usage of technology to benefit the public, through sharing of information and functionality while protecting user privacy and data. He received his M.S. in Computer Science from Northeastern University.
                                </p>
                            </div>
                        </div>
                        <br/>
<!--                        <h4>Visiting students</h4>-->
                        <h4>Friends of the lab</h4>
                        <h5>Rahul Gupta</h5>
                        <h6>Graduate student</h6>
                        <div class="row">
                            <div class="col-sm-4">
                                <img src="people/RahulGupta.jpg" width="100%" />
                            </div>
                            <div class="col-sm-8">
                                <p>
                                    Rahul is an MD-PhD student in the Harvard-MIT Health Sciences and Technology program, and is currently doing his PhD in Genetics and Genomics at Harvard. He completed his undergraduate work in Chemical Engineering and Biology at the University of Pennsylvania in 2016. Co-mentored by Drs. Ben Neale and Vamsi Mootha, Rahul's work is focused on using human genetics to better understand perturbed pathways and complex disease mechanism, with a focus on mitochondrial function and dysfunction.
                                </p>
                            <p>
                                <a href="https://twitter.com/rahulg603?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">@rahulg603</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
<a class="github-button" href="https://github.com/rahulg603" aria-label="Follow @rahulg603 on GitHub">@rahulg603</a>
                            </p>
                            </div>
                        </div>
<!--                        <div class="row">-->
<!--                            <div class="col-sm-4">-->
<!--                                <img src="people/" width="300px" />-->
<!--                            </div>-->
<!--                            <div class="col-sm-8">-->
<!--                                <p>-->
<!--                                    Blurb.-->
<!--                                </p>-->
<!--                                <p>-->
<!--                                    <a href="https://twitter.com/konradjk?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">@konradjk</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>-->
<!--            <a class="github-button" href="https://github.com/konradjk" aria-label="Follow @konradjk on GitHub">@konradjk</a>-->
<!--                                </p>-->
<!--                            </div>-->
<!--                        </div>-->
                        <br/>
                        <h4>Alumni</h4>
                        <div class="row">
                            <ul>
                                <li>
                                    Ulrik Stoltze (visiting PhD student, University Hospital Copenhagen)
                                </li>
                                <li>
                                    Adrian Janucik (visiting Masters student, Medical Univ of Bialsytok)
                                </li>
                                <li>
                                    Hannah Nicholls (visiting Masters student, QMUL)
                                </li>
                            </ul>
                        </div>
                    </div>
                    <br/>
                    <div id="philosophy">
                        <h3>Core tenets</h3>
                        <p>
                            We are committed to training the next generation of computational scientists. A crucial component of this training is building a collaborative spirit in the team, promoting a happy and healthy environment where everyone can excel, and ensuring that we move the science forward together in a rigorous fashion.
                        </p>
                        <h4>What we do</h4>
                            <p>
                                Our core mission involves the use of massive datasets to learn about human disease and the biology of the genome. We value high quality data and code, ensuring reproducibility, and openness to advance human genetics.
                            </p>
                            <h5>Big data</h5>
                            <p>
                                The onslaught of genetic data has arrived, and we are fortunate to work in a time when massive data sizes enable rigorous approaches and robust statistics. However, these data volumes also require special handling: at the Karczewski lab, we take a "cloud-first" approach to computational biology. As we are managing datasets in the 1TiB range (and some crossing the 100TiB threshold), scalable computation is a must. We value our mutually-beneficial relationship with the <a href="https://hail.is/">Hail team</a>, where we build our pipelines in Hail and feed back our progress or issues, and they enable our ideas for massive-scale analysis.
                            </p>
                            <h5>High quality data</h5>
                            <p>
                                One of the cornerstones of efficient scientific progress in a field is the veracity of the literature, which creates confidence and builds trust. On the other hand, high dimensional datasets that represent millions to quadrillions of measurements have an inherent error rate. Understanding the error modes for each dataset, developing methods to address them, and faithfully reporting our results, including all raw data and code (see Reproducibility below), is a crucial step in ensuring high quality. This is an iterative process that occasionally takes longer than we'd like, but the end result is something we can all be proud of.
                            </p>
                            <h5>Reproducibility</h5>
                            <p>
                                Similarly, as our data grows and computational pipelines become more complex, having a foundation of public code creates a record of our analytical approach. Partially for the community, and partially for ourselves, the ability to reproduce each step of an analysis promotes code reusability and pays dividends on the inevitable need to rerun code after peer review, dataset updates, or that one additional QC step upstream. Additionally, public reproducible code leads to fewer mistakes due to out-of-sync intermediate files, and ensures that even when mistakes do occur, it is easy to document what steps are affected and to what degree, allowing for rapid correction of the scientific record if needed.
                            </p>
                            <h5>Open science</h5>
                            <p>
                                The <a href="https://www.atgu.mgh.harvard.edu/atgu-publication-policy/">publication policy</a> at ATGU pledges that in the spirit of rapid open science, we will submit all manuscripts to a preprint server at the time of journal submission. Further to this, the large-scale datasets that we manage are property of the community, not the individual researcher that happened to aggregate them or run a particular analysis on them. As stewards of these data, we commit to release intermediate products of the data on completion of quality control rather than the time of publication, if allowed by regulatory bodies. This may include hosting datasets, primarily for initial releases of data, or browser frameworks to enable exploration of the data.
                            </p>
                        <h4>How we do it</h4>
                            <p>
                                Equally important to the science we advance is the manner in which we perform the research and the development of each lab member. As every career level and trajectory will be different, I encourage each lab member to reach out to define a set of goals, so that we can work together to achieve them. Lab members are expected to pursue rigorous research and effective dissemination of this research, which often involves publications, though other forms of communication, including code release and browser development are also encouraged.
                            </p>
                            <h5>Collaboration/new directions</h5>
                            <p>
                                The ATGU and the Broad Institute are highly dynamic and collaborative environments, where we work with experts in computational biology, statistical genetics, population genetics, scalable computing, medical genetics, and more. As all trainees build a research program, we encourage discussion of the ideas, methods, and results with those in the local and broader environments, in order to advance the science as effectively as possible. Credit is infinitely divisible, and collaborations lead to new ideas and additional publications for all involved. Trainees should feel empowered to form collaborations, and I am happy to advise how to navigate these. Similarly, pursuing new directions that may or may not be related to your primary projects can be a valuable endeavor, and I encourage spending 10-20% of your time on average on exploring new avenues, which may also include consulting for other companies. Feel free to discuss these directions with me so that we can ensure your time is spent effectively.
                            </p>
                            <h5>Work life balance</h5>
                            <p>
                                While our work is important, I believe that a happy and healthy team is an effective team. To this end, I encourage each member of the team to identify and adopt their most efficient style, and to respect the choices of others. I expect an amount of work requisite with what is written in your contract, but understand that different stages of work and life may require different considerations. To this end, I am flexible on working hours, and though Slack messages may come at all times or on all days, no one should feel pressured to respond to messages outside of their working hours. Occasional exceptions may arise, such as a lead up to conferences like ASHG, but in these cases, taking a long-needed break after the event is encouraged.
                            </p>
                            <h5>Communicating science</h5>
                            <p>
                                A critical aspect of being a successful scientist in the modern era is the ability to communicate our research to others in our field, other scientists, and the public. I will provide opportunities for lab members to present their research internally and externally, and I commit to helping each individual craft a narrative for their research as they present to the public. I also encourage public discussion of research before publication (see Open Science, above), as well as teaching opportunities as desired.
                            </p>
                            <h5>Mentoring</h5>
                            <p>
                                For individuals further along in their training, mentoring junior researchers is a key component to a well-rounded training. I am happy to discuss an arrangement that suits everyone's research questions and career trajectories.
                            </p>
                    </div>
                    <div id="publications">
                    <h3>Selected Publications</h3>
                    <p>
                        Full list available at <a href="https://scholar.google.com/citations?user=fuAzo2IAAAAJ">Google Scholar</a>:
                    </p>
                    <p>
                        <b>Karczewski KJ</b>, Francioli LC, Tiao G, Cummings BB, Alföldi J, Wang Q, et al.,
                        <a href="https://www.biorxiv.org/content/10.1101/531210v3">
                            "The mutational constraint spectrum quantified from variation in 141,456 humans."
                        </a><i>Nature</i>.
                        2020 May 27. doi: 10.1038/s41586-020-2308-7.
                        Flagship article for the <a href="https://www.nature.com/nature/volumes/581/issues/7809">gnomAD package</a>.
                        See <a href="#press-gnomad">Press</a> below.
                    </p>
                    <p>
                        <b>Karczewski KJ</b>, and Martin AR.
                        <a href="https://www.biorxiv.org/content/10.1101/531210v3">
                            "Analytic and Translational Genetics."
                        </a><i>Annu Rev Biomedical Data Sci</i>.
                        2020. doi: 10.1146/annurev-biodatasci-072018-021148.
                    </p>
                    <p>
                        <b>Karczewski KJ</b> and Snyder M.
                        <a href="https://www.nature.com/articles/nrg.2018.4">
                            "Integrative Omics for Health and Disease."
                        </a><i>Nat Rev Genet</i>.
                        2018 May; 19(5):299-310. doi: 10.1038/nrg.2018.4.
                    </p>
                    <p>
                        <b>Karczewski KJ</b>, Weisburd B, Thomas B, Ruderfer DM, Kavanagh D, Hamamsy T, et al.,
                        <a href="https://academic.oup.com/nar/article/45/D1/D840/2572071">
                            "The ExAC Browser: Displaying reference data information from over 60,000 exomes."
                        </a><i>Nucleic Acids Res</i>.
                        2017 Jan 4; 45(D1):D840-D845. doi: 10.1093/nar/gkw971. Epub 2016 Nov 28.
                        (<i>bioRxiv</i>. doi: 10.1101/070581. 2016 Aug 19.)
                    </p>
                    <p>
                        Lek M, <b>Karczewski KJ</b>*, Minikel EV*, Samocha KE*, Banks E, Fennell T, et al.,
                        <a href="https://www.nature.com/articles/nature19057">
                            "Analysis of protein-coding genetic variation in 60,706 humans."
                        </a><i>Nature</i>.
                        2016 Aug 17; 536(7616):285-91. doi: 10.1038/nature19057.
                        (<i>bioRxiv</i>. doi: 10.1101/030338. 2015 Oct 30).
                    </p>
                    <p>
                        <b>Karczewski KJ</b>, Snyder M, Altman RB, Tatonetti NP.
                        <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004122">
                            "Coherent functional modules improve transcription factor target identification, cooperativity prediction, and disease association."
                        </a><i>PLoS Genetics</i>.
                        10(2): e1004122. doi:10.1371/journal.pgen.1004122.s012
                    </p>
                    <p>
                        <b>Karczewski KJ</b>*, Dudley JT*, Kukurba KR, Chen R, Butte AJ, Montgomery SB, Snyder M.
                        <a href="https://www.pnas.org/content/110/23/9607.long">
                            "Systematic functional regulatory assessment of disease-associated variants."
                        </a><i>Proc Natl Acad Sci U S A</i>.
                        Epub 2013 May 20. doi: 10.1073/pnas.1219099110.
                    </p>
                    <p>
                        Dudley JT and <b>Karczewski KJ</b>.
                        <a href="https://global.oup.com/academic/product/exploring-personal-genomics-9780199644490">Exploring Personal Genomics.</a>
                        January 2013. Oxford University Press.
                    </p>
                    <p>
                        <b>Karczewski KJ</b>*, Tirrell RP*, Tatonetti NP, Dudley JT, Cordero P, Salari K, et al.,
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/pmid/22174289/">
                            "Interpretome: A Freely Available, Modular, and Secure Personal Genome Interpretation Engine."
                        </a><i>Pac Symp Biocomput</i>.
                        Epub 2011 Oct 25. 17:339-350(2012).
                    </p>
                    <p>
                        <b>Karczewski KJ</b>, Tatonetti NP, Landt SG, Yang X, Slifer T, Altman RB, Snyder M.
                        <a href="https://www.pnas.org/content/108/32/13353.long">
                            "Cooperative Transcription Factor Associations Discovered using Regulatory Variation."
                        </a><i>Proc Natl Acad Sci U S A</i>.
                        2011 Aug 9;108(32):13353-8. doi: 10.1073/pnas.1103105108. Epub 2011 Jul 26.
                    </p>
                    </div>
                    <div id="press">
                    <h3>Press</h3>
                    <h5 id="press-gnomad">gnomAD</h5>
                    <ul>
                        <li>
                            Full <a href="https://www.nature.com/immersive/d42859-020-00002-x/index.html">collection of gnomAD papers</a>
                            in Nature, Nature Medicine, and Nature Communications.
                        </li>
                        <li>
                            Coverage in
                            <a href="https://cosmosmagazine.com/science/biology/genetic-library-grows/">
                                Cosmos magazine</a>,
                            <a href="https://www.elmundo.es/ciencia-y-salud/ciencia/2020/05/27/5ece795afdddffc5bd8b45f9.html">
                                El mundo</a>,
                            <a href="https://medicalxpress.com/news/2020-05-genetic-database-rare-disease-clues.html">
                                Medical Xpress</a>, and
                            <a href="https://www.genomeweb.com/sequencing/gnomad-consortium-details-range-human-genetic-variation-large-dataset">genomeweb</a>.
                        </li>
                        <li>
                            Other press coverage from
                            <a href="https://leht.postimees.ee/6983415/teadusrevolutsioon-inimese-geneetika-uurimine-tegi-tohutu-arengusammu">
                                Estonia</a>,
                            <a href="https://www.tekcrispy.com/2020/05/28/base-datos-genetica-enfermedades-raras/">
                                Spain</a>, and
                            <a href="https://www.france24.com/en/20200527-scientists-unravel-secrets-from-the-faults-in-our-genes">
                                France</a>.
                        </li>
                        <li>
                            <a href="https://www.nature.com/articles/d41586-020-01485-4">News and Views</a> and
                            <a href="https://www.nature.com/articles/d41586-020-01551-x">Editorials</a> in Nature.
                        </li>
                        <li>
                            Research highlight in <a href="https://www.nature.com/articles/s41576-020-0255-7">Nature Reviews Genetics</a>.
                        </li>
                    </ul>
                    </div>
                    <div id="public_projects">
                    <h3>Public Projects</h3>
                    <p>
                        <a href="https://gnomad.broadinstitute.org">gnomAD</a>: The Genome Aggregation Database, dataset and browser.
                    </p>
                    <p>
                        <a href="https://genebass.org">Genebass</a>: Rare variant associations for >3k phenotypes in the UK Biobank.
                    </p>
                    <p>
                        <a href="https://github.com/konradjk/loftee">LOFTEE</a>: loss-of-function variation annotation.
                    </p>
                    <p>
                        <a href="https://hail.is">Hail</a> (contributor): open-source library for large-scale data analysis.
                    </p>
                    </div>
                    <div id="contact">
                    <h3>Contact</h3>
                    <p>
                        Email: <a href="mailto:konradjkarczewski@gmail.com">konradjkarczewski@gmail.com</a>
                    </p>
                    <p>
                        Follow me on <a href="https://twitter.com/konradjk">Twitter</a>,
                        <a href="http://github.com/konradjk/">Github</a>,
                        or see my <a href="http://amazon.com/author/konradjkarczewski">Amazon Author Page</a>.
                    </p>
                    </div>
                </div>
                <div class="col-sm-4">
                    <a class="twitter-timeline" href="https://twitter.com/konradjk/timelines/1167023909310488576?ref_src=twsrc%5Etfw">konradjk - Curated tweets by konradjk</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
                </div>
            </div>
        </div>
    </div>
</div>

    <script type="text/javascript">
        $(".nav-item").on("click", function(){
           $("li.nav-item.active").removeClass("active");
           $(this).addClass("active");
        });
    </script>
<!-- Place this tag in your head or just before your close body tag. -->
<script async defer src="https://buttons.github.io/buttons.js"></script>
<script src="https://platform.linkedin.com/badges/js/profile.js" async defer type="text/javascript"></script>
</body>
</html>
